### PR TITLE
[Travis] Fix tests, move to newer version of phpunit

### PR DESCRIPTION
**Pull request by andrerom**
Some cleanup here too in order to align with other repos.

### CODE REVIEW

File: Tests/SystemInfo/Collector/ConfigurationSymfonyKernelSystemInfoCollectorTest.php [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 use EzSystems\EzSupportToolsBundle\SystemInfo\Collector\ConfigurationSymfonyKernelSystemInfoCollector;
 use EzSystems\EzSupportToolsBundle\SystemInfo\Value\SymfonyKernelSystemInfo;
-use PHPUnit_Framework_TestCase;
+use PHPUnit\Framework\TestCase as PHPUnit_Framework_TestCase;
```
Comment (alongosz) [inline Tests/SystemInfo/Collector/ConfigurationSymfonyKernelSystemInfoCollectorTest.php]: Wouldn't it be simpler to change `extends PHPUnit_Framework_TestCase` into `extends TestCase`? Just one line more ;)

Comment (andrerom) [reply Tests/SystemInfo/Collector/ConfigurationSymfonyKernelSystemInfoCollectorTest.php]: sure nitty :)

Comment (andrerom) [reply Tests/SystemInfo/Collector/ConfigurationSymfonyKernelSystemInfoCollectorTest.php]: Fixed in e63eadc
Also bumped php-cs-fixer in 8438f57 so runs on PHP 7.3 works

File: .travis.yml [hunk 2]
```diff
@@ -1,29 +1,39 @@
...
+    fast_finish: true
     include:
         - php: 5.6
         - php: 7.0
         - php: 7.1
           env: CHECK_CS=true
+        - php: 7.2
+        - php: 7.3
 
 # test only master (+ Pull requests)
```
Comment (glye) [inline .travis.yml]: Not "only master" anymore given change below, right?

Comment (andrerom) [reply .travis.yml]: indeed, fixed in 
a948ba1

**Pull request merged**